tools/cli/adminCommands.go

```diff
@@ -38,7 +37,7 @@ import (
 	historypb "go.temporal.io/temporal-proto/history/v1"
 
 	"github.com/temporalio/temporal/.gen/proto/adminservice/v1"
-	enumsgenpb "github.com/temporalio/temporal/.gen/proto/enums/v1"
+	enumspb "github.com/temporalio/temporal/.gen/proto/enums/v1"
```

Comment:
Here are proto package alias rules:
1. `*pb` alias is always used (even if it is not needed) for all packages from `go.temporal.io/temporal-proto` besides services (`workflowservice`).
2. `*genpb` alias is always used (even if it is not needed) for all packages from `github.com/temporalio/temporal/.gen/` besides services (`historyservice`, `matchingservice`,`admindservice`).

So please revert this back to `enumsgenpb`.